src/js/chat/templates.js

```diff
@@ -188,6 +188,21 @@ var bttvMessageTokenize = exports.bttvMessageTokenize = function(sender, message
     return tokenizedString.join(' ');
 };
 
+exports.bttvElementTokenize = function(senderEl, messageEl) {
+    var newTokens = [];
+    var tokens = $(messageEl).contents();
+    var sender = $(senderEl).text().trim().toLowerCase();
+    for (var i = 0; i < tokens.length; i++) {
+        if (tokens[i].nodeType === Node.TEXT_NODE) {
```

Comment:
where is "Node" defined? is it on window? can we reference `window.` in that case to make it clearer